pkg/commands/git_commands/commit_loader.go

```diff
@@ -357,7 +357,7 @@ func (self *CommitLoader) setCommitMergedStatuses(refName string, commits []*mod
 		if strings.HasPrefix(ancestor, commit.Sha) {
 			passedAncestor = true
 		}
-		if commit.Status != models.StatusPushed {
+		if commit.Status != models.StatusPushed && commit.Status != models.StatusUnpushed {
```

Comment:
I didn't add any test for this, as I wasn't sure what kind of test coverage is appropriate for this.